_persons/thanhson.md

```diff
@@ -0,0 +1,28 @@
+---
+# use person if you want to have a personal website created
+layout: person
+first_name: Thanh Son
+last_name: Nguyen
+key: thanhson
```

Comment:
Your key should be your last name. So you should change that everywhere. Everything else looks good.

Reply:
I changed it